ui/ducks/send/send.js

```diff
@@ -442,6 +451,10 @@ export const initializeSendState = createAsyncThunk(
       gasPrice = getGasPriceInHexWei(
         gasFeeEstimates.medium.suggestedMaxFeePerGas,
       );
+    } else {
```

Comment:
@PeterYinusa @adonesky1 This is the one change I made since last review

Reply:
👍